samples/secret/secret.yaml

```diff
@@ -9,7 +9,7 @@
 # WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
 # See the License for the specific language governing permissions and
 # limitations under the License.
-# To create this secret use: kubectl create secret generic powermax-config -n powermax --from-file=config=./secret.yaml
+# To create this secret use: kubectl create secret generic powermax-creds -n powermax --from-file=config=./secret.yaml
```

Comment:
Any impact on backward compatibility. Ex: Someone has already created a secret `powermax-config` and then try to upgrade to latest driver?

Reply:
This is for the next release so any upgrades of this release will already have the previous secret. The current instructions is definitely wrong.

Reply:
I also see powermax-creds being used in 10.2. https://dell.github.io/csm-docs/v3/deployment/csmoperator/drivers/powermax/. This seems to be a long overdue update.